File: AD-HOC/2388/main.c
#include <stdio.h>

int main() {
    int n, t, v, distance = 0;
    scanf("%d", &n);
    for (int times = 0; times < n; times++) {
        scanf("%d %d", &t, &v);
        distance += t * v;
    }
    printf("%d\n", distance);
    return 0;
}

File: AD-HOC/2388/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int distance = 0;
        int n = scanner.nextInt();
        for (int times = 0; times < n; times++) {
            int t = scanner.nextInt();
            int v = scanner.nextInt();
            distance += t * v;
        }
        System.out.println(distance);
        scanner.close();
    }
}
